lib/api.js

```diff
@@ -54,9 +54,12 @@ function API(opts) {
     log.setLevel('info');
   }
 };
-
 util.inherits(API, events.EventEmitter);
 
+API.sjclOpts = {
```

Comment:
Maybe call this privateKeyEncryptionOpts? I think it would help understanding the purpose of this config.